curriculum/challenges/english/07-scientific-computing-with-python/learn-encapsulation-by-building-a-projectile-trajectory-calculator/6633bfcef57e1d70cc3142c6.md

```diff
@@ -0,0 +1,182 @@
+---
+id: 6633bfcef57e1d70cc3142c6
+title: Step 20
+challengeType: 20
+dashedName: step-20
+---
+
+# --description--
+
+Finally, make the final output a multiline string. It should start with a newline character and end with a newline character.
+
+Save it in `graph`.
+
+```py
+
+⊣     ∙       
+⊣  ∙∙∙ ∙∙∙    
+⊣ ∙       ∙   
+⊣∙         ∙  
+⊣           ∙ 
+⊣            ∙
+⊣             
+ ⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤⊤
+
+```
+
+Return `graph`.
+
+# --hints--
+
+The method `create_trajectory` should return `graph`.
+
+```js
+({
+    test: () => runPython(`
+    assert _Node(_code).find_class('Graph').find_function('create_trajectory').has_return('graph'), "return graph missing"
+    `)
+})
+```
```

Comment:
If the exact value from function is tested (next hint) I think it's fine to not check how the returned _thing_ is named in function. This likely apply to couple other steps.